packages/flutter/lib/src/widgets/page_view.dart

```diff
@@ -79,8 +79,7 @@ import 'viewport.dart';
 ///             Container(
 ///               color: Colors.red,
 ///               child: Center(
-///                 child: RaisedButton(
-///                   color: Colors.white,
+///                 child: ElevatedButton(
```

Comment:
Not sure why we were specifying `color: Colors.white` before; it's OK to leave the parameter out now? Here and below.

Reply:
I think so. I just checked this in dartpad and it looks fine without the white color for `RaisedButton`:

https://dartpad.dev/5c0e3ff90da4da543a2d850d1690f259

Not sure why. it was specified in first place.

Reply:
Glad to hear that it wasn't needed.